system_tests/tests/aldn1000.py

```diff
@@ -194,9 +194,14 @@ def test_GIVEN_device_connected_WHEN_get_status_THEN_device_status_returned(self
 
     @skip_if_recsim("Unable to use lewis backdoor in RECSIM")
     def test_GIVEN_device_not_connected_WHEN_get_error_THEN_alarm(self):
-        self._lewis.backdoor_set_on_device('connected', False)
-        self.ca.set_pv_value("STATUS.PROC", 1)
-        self.ca.assert_that_pv_alarm_is('STATUS', ChannelAccess.Alarms.INVALID, timeout=5)
+        self.ca.assert_that_pv_alarm_is('STATUS', ChannelAccess.Alarms.NONE, timeout=5)
+
+        with self._lewis.backdoor_simulate_disconnected_device():
+            self.ca.set_pv_value("STATUS.PROC", 1)
+            self.ca.assert_that_pv_alarm_is('STATUS', ChannelAccess.Alarms.INVALID, timeout=5)
+            
+        # Assert alarms clear on reconnection
+        self.ca.assert_that_pv_alarm_is('STATUS', ChannelAccess.Alarms.NONE, timeout=5)
```

Comment:
How do you feel about placing this into a separate private method or new test entirely?